util/omrutil/AtomicFunctions.cpp

```diff
@@ -60,3 +60,13 @@ issueWriteBarrier(void)
 {
 	VM_AtomicSupport::writeBarrier();
 }
+ 
+void addAtomic(volatile uintptr_t *address, uintptr_t addend)
```

Comment:
Nit pick: to be consistent with the coding standard and the rest of the file, put the return type on it's own line.